public/app/plugins/panel/canvas/globalStyles.ts

```diff
@@ -1,11 +1,248 @@
 import { css } from '@emotion/react';
 
-import { GrafanaTheme2 } from '@grafana/data';
+import { GrafanaTheme } from '@grafana/data';
 
-export function getGlobalStyles(theme: GrafanaTheme2) {
+export function getGlobalStyles(theme: GrafanaTheme) {
```

Comment:
we should really use theme2 unless we are totally stuck or migrating old stuff.  Note you can get to theme1 from the theme2 object :)